<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>993</width>
    <height>726</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>科學計算程式</string>
  </property>
  <property name="styleSheet">
   <string notr="true">    /* 背景與全局文字 */
    QMainWindow, QWidget {
        background-color: #f5f6f7;
        font-family: &quot;Segoe UI&quot;, &quot;Microsoft JhengHei&quot;;
        font-size: 10pt;
        color: #333333;
    }

    /* 分頁標籤樣式 */
    QTabWidget::pane { 
        border: 1px solid #dcdcdc;
        background: white;
    }
    QTabBar::tab {
        background: #e1e1e1;
        padding: 8px 20px;
        margin-right: 2px;
        border-top-left-radius: 4px;
        border-top-right-radius: 4px;
    }
    QTabBar::tab:selected {
        background: white;
        border-bottom: 2px solid #0078d7; /* 底部藍色條，現代感重點 */
        font-weight: bold;
    }

    /* 群組框 (GroupBox) */
    QGroupBox {
        border: 2px solid #e0e0e0;
        border-radius: 8px;
        margin-top: 1.5ex;
        font-weight: bold;
        background-color: white;
    }
    QGroupBox::title {
        subcontrol-origin: margin;
        subcontrol-position: top center;
        padding: 0 10px;
        color: #0078d7;
    }

    /* 輸入框 (LineEdit) */
    QLineEdit {
        border: 1px solid #cccccc;
        border-radius: 4px;
        padding: 4px;
        background: #ffffff;
    }
    QLineEdit:focus {
        border: 2px solid #0078d7; /* 點擊時變藍框 */
    }
    QLineEdit[readOnly=&quot;true&quot;] {
        background-color: #f0f0f0;
        color: #888888;
    }

    /* 下拉選單 (ComboBox) */
    QComboBox {
        border: 1px solid #cccccc;
        border-radius: 4px;
        padding: 4px 18px 4px 4px;
        background: white;
    }
    QComboBox::drop-down {
        border: none;
    }</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>800</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>工程記號換算</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>170</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string> 單位換算矩陣 </string>
      </property>
     </widget>
     <widget class="QTableWidget" name="matrixTable">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>190</y>
        <width>1071</width>
        <height>331</height>
       </rect>
      </property>
      <property name="rowCount">
       <number>9</number>
      </property>
      <property name="columnCount">
       <number>9</number>
      </property>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>60</y>
        <width>211</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>輸入</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>191</width>
         <height>67</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="frameShape">
           <enum>QFrame::Shape::Box</enum>
          </property>
          <property name="text">
           <string>數值</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_4">
          <property name="frameShape">
           <enum>QFrame::Shape::Box</enum>
          </property>
          <property name="text">
           <string>記號</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="Input_lineEdit"/>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="Input_comboBox"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="output_groupBox">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>60</y>
        <width>221</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>輸出</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
      <widget class="QWidget" name="layoutWidget1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>201</width>
         <height>67</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="frameShape">
           <enum>QFrame::Shape::Box</enum>
          </property>
          <property name="text">
           <string>數值</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_6">
          <property name="frameShape">
           <enum>QFrame::Shape::Box</enum>
          </property>
          <property name="text">
           <string>記號</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="Outputput_lineEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="output_comboBox"/>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>基本SMD 代碼解碼</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>電阻分壓計算</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>LED 限流電阻計算</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>走線電流計算</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>貫孔電流計算</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>993</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>工具</string>
    </property>
    <addaction name="actioncalc"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>說明</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actioncalc">
   <property name="text">
    <string>計算機(not yet)</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
